sslcert/tls.crt

```diff
@@ -0,0 +1,18 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Remove these certs.  Add to .gitignore so we don't merge accidentally with future PRs.

Reply:
done